EventHubs/src/SumoEventHub.csproj

```diff
@@ -6,19 +6,12 @@
     <PublishOutputPathContents>true</PublishOutputPathContents>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Gcm.AzureRM.PowerShell.Deployment" Version="1.*" />
-    <PackageReference Include="Gcm.ConfigurationLoader" Version="3.0.0-netstandard0039" />
+    <PackageReference Include="Gcm.AzureAz.PowerShell" Version="0.*" />
```

Comment:
is this in the trello board? if so, update it!

Reply:
Assumed it was earlier and upgraded without checking the board, but this repo does not show up in the trello board.  should i add it? and do you mean update to 1.* version for AzureAz?

Reply:
yeah add it. might as well use 1.* too